config/jobs/kubernetes/sig-apps/sig-apps-config.yaml

```diff
@@ -1,30 +1,4 @@
 periodics:
-- interval: 30m
-  name: ci-kubernetes-e2e-gce-taint-evict
-  labels:
-    preset-service-account: "true"
-    preset-k8s-ssh: "true"
-  spec:
-    containers:
-    - args:
-      - --timeout=80
-      - --bare
-      - --scenario=kubernetes_e2e
-      - --
-      # TODO(gmarek): Enable this when we've split 1.2 tests into another project.
-      - --env=CONTROLLER_MANAGER_TEST_ARGS=--feature-gates=TaintBasedEvictions=true --enable-taint-manager=true
-      - --extract=ci/latest
-      - --gcp-node-image=gci
-      - --gcp-zone=us-west1-b
-      - --provider=gce
-      - --test_args=--ginkgo.focus=\[Feature:TaintEviction\] --minStartupPods=8
-      - --timeout=50m
-      image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220413-ebb00ad7a6-master
-
-
-  annotations:
-    testgrid-dashboards: google-gce
```

Comment:
aside: really need to nuke most of these google-* dashboards entirely, they're really not meaningful at this point

Reply:
for sure!